packages/manager/modules/web-paas/src/components/additional-option/translations/Messages_fr_FR.json

```
@@ -0,0 +1,8 @@
+{
+  "web_paas_offer_additional_option_title_staging_environment": "Ajouter des environnements de test",
+  "web_paas_offer_additional_option_title_storage": "Ajouter du stockage",
+  "web_paas_offer_additional_option_title_user_license": "Ajouter des licences utilisateur",
+  "web_paas_offer_additional_option_price_desc_staging_environment": "/mois",
+  "web_paas_offer_additional_option_price_desc_storage": "/environnement/mois",
+  "web_paas_offer_additional_option_price_desc_user_license": "/mois"
```

Could be nice to have a value 
```suggestion
  "web_paas_offer_additional_option_price_desc_user_license": "{{price }}/mois"
```
in case the way it translates has a different order

---

Have different styles for price and `/month`. So have added translation just for `/month` and displaying price using `<h4>` tag here